src/Quasar/Mount/SparkHDFS.purs

```diff
@@ -54,7 +54,7 @@ fromJSON ∷ Json → Either String Config
 fromJSON
   = fromString
   <=< (_ .? "connectionUri")
-  <=< (_ .? "spark")
```

Comment:
Right, but the comment is suggesting that `"spark"` may be incorrect. Should it not be `"spark-hdfs"` (I realize it shouldn't be SparkLocal) to match the decoder?